jdaviz/configs/imviz/tests/test_catalogs.py

```diff
@@ -158,14 +158,25 @@ def test_plugin_image_with_result(self, imviz_helper, tmp_path):
         assert imviz_helper.viewers['imviz-0']._obj.state.y_min == -0.5
         assert imviz_helper.viewers['imviz-0']._obj.state.y_max == 1488.5
 
+        # Re-populate the table with a new search
+        with pytest.warns(ResourceWarning):
+            catalogs_plugin.search(error_on_fail=True)
+        # Ensure at least one row is selected before zooming
+        catalogs_plugin.table.selected_rows = [catalogs_plugin.table.items[0]]
+        assert len(catalogs_plugin.table.selected_rows) > 0
+
         # set 'padding' to reproduce original hard-coded 50 pixel window
         # so test results don't change
         catalogs_plugin.zoom_to_selected(padding=50 / 2048)
 
-        assert imviz_helper.viewers['imviz-0']._obj.state.x_min == 858.24969
-        assert imviz_helper.viewers['imviz-0']._obj.state.x_max == 958.38461
-        assert imviz_helper.viewers['imviz-0']._obj.state.y_min == 278.86265
-        assert imviz_helper.viewers['imviz-0']._obj.state.y_max == 378.8691
+        assert_allclose(
+            imviz_helper.viewers['imviz-0']._obj.state.x_min, 1022.57570000, atol=0.1)
+        assert_allclose(imviz_helper.viewers['imviz-0']._obj.state.x_max,
+                        1122.5757, atol=0.1)
```

Comment:
Consistent styling with the other 3 checks to make it easier to read.
```suggestion
        assert_allclose(
            imviz_helper.viewers['imviz-0']._obj.state.x_max, 1122.5757, atol=0.1)
```